.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -56,6 +49,14 @@ class LoadTestsBuilder {
     }
   }
 
+  static String setContextualDatasetName(String baseName, TriggeringContext triggeringContext) {
```

Comment:
I think this method name doesn't seem self-explanatory, maybe you can think of something more suggesting that we change BQ dataset based on triggered context eg. `setBQMetricsDatasetName` or maybe `setBQDatasetNameByTriggerContext`?

Reply:
`setBQDatasetNameByTriggerContext ` sgtm

Reply:
Alright, I will change this name to `setBQDatasetNameByTriggerContext`